### ISSUE TITLE
Displaying more rows disrupts display of reference texts

### ISSUE THREAD

**Issue by lrsallee** [unfoldingWord/tc-create-app#49]
I tried editing Titus in my fork of tN. I completed the first section of 25 rows and then changed to 100 row display successfully (as I remember). However when I moved to the next 100 rows, the scripture pane stopped updating and remained blank.

4 second max.

**Comment by lrsallee** [comment]
OK, I take that back. It took so long that I went through the process of creating an issue before it finally appeared. This for a short 3-chapter book.

**Comment by lrsallee** [comment]
When the references finally displayed, many of them did not show the highlighting, or only showed it in the ULT but not the other texts. I waited 10+ minutes with no change. The first verse with the missing highlights was Titus 2:13, but they were scattered through the rest of the chapter.

**Comment by lrsallee** [comment]
![image](https://user-images.githubusercontent.com/5585656/72019388-9acdf380-3237-11ea-8b50-6fd7ebb4583e.png)

**Comment by mandolyte** [comment]
@klappy on a hunch, in my local copy of tCC with the hooks/context features, I changed the config to:
```js
        const serverConfig = { 
          server: SERVER_URL,
          cache: {
            maxAge: 1 * 1 * 1 * 60 * 10000, // override cache to 1 minute
          },
        };
```
Namely, 1000 to 10000 (expecting this to mean 10 minutes). It did seem to improve things. And I wondered how to interpret "maxAge". In particular, typically, each of the ones in "1 * 1 * 1" indicate a different unit of duration being multiplied together. But not sure that's right... 

So does the maxAge property work? Thanks!

**Comment by elsylambert** [comment]
translationCore Create - v1.0.0-beta.2. Smaller books opens pretty fast up to 100 rows. For larger NT books, the rows opens faster but PSV takes longer to load which would be addressed by rcl-#23.
